src/include/ndpi_typedefs.h

```diff
@@ -1055,6 +1055,11 @@ typedef enum {
   */
   CUSTOM_CATEGORY_ANTIMALWARE      = 105,
   
+  /*
+    Crypto Currency e.g Bitcoin, Litecoin, Etherum ..et.
+  */
+  NDPI_PROTOCOL_CATEGORY_CRYPTO_CURRENCY = 106,
```

Comment:
Not sure if this category is in conflict of `NDPI_PROTOCOL_CATEGORY_MINING`.
If not, what is the difference?

Reply:
* Not each crypto exchange is a mining, it could be a normal transaction. 
* so we need to differentiate between normal crypto protocols such as when someone is just sending or receiving transactions and when he is mining. 
* So in a previous email we agreed to split this part and have two different categories: 
            - a cryptocurrency that detects normal exchanges 
            - Mining that detects mining protocols e.g. stratum.

Reply:
@mmaatuq, could you add a brief version of this difference in `doc/protocols.rst` under a "Notes" section in "Bitcoin", please? [see SRTP for an example]

I consider myself a guy with an average knowledge of network protocols but I didn't know this difference before reading it in your mail. Therefore I think it might be useful for any users

Reply:
Yeah sure, will do.

Reply:
@utoni  you are welcome.